### PR TITLE
Drop Django 2.2 support.

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (Linkid) [inline .pre-commit-config.yaml:7]: Python 3.7 is the oldest supported Python version and is still supported by Django 3.2. So, I don't think it is necessary for now to update it.

File: CHANGELOG.md
Comment (Linkid) [inline CHANGELOG.md:6]: Maybe precise that `django_jsonfield_backport` is not required anymore ?

**Review by Linkid — CHANGES_REQUESTED** [review]
Django 2.2 is still supported until the end of the month (April 2022) :p. Maybe we can wait until May 2022.

File: .pre-commit-config.yaml
Comment (hramezani) [reply .pre-commit-config.yaml:7]: I think this `language_version` here determines the python version to run the black on it.
As it was `py38` on tox, I've changed it to `python3.7`

**Comment by hramezani** [comment]
> Django 2.2 is still supported until the end of the month (April 2022) :p. Maybe we can wait until May 2022. (A new patch for Django 2.2 is scheduled for April 11, 2022)

Yes, This is just for preparation. we can wait until May 2022 and then merge it.

**Comment by hramezani** [comment]
@Linkid I think we can merge this one now and prepare a new release soon.
What do you think?

**Comment by Linkid** [comment]
It is ok for me, yes. But could you add to the changelog that `django_jsonfield_backport` is not required anymore, please?

File: CHANGELOG.md
Comment (hramezani) [reply CHANGELOG.md]: Done!

**Comment by hramezani** [comment]
> Good idea, yes. But could you add to the changelog that `django_jsonfield_backport` is not required anymore, please?

Sorry, I forgot about it :man_facepalming:. Done!

**Review by Linkid — APPROVED** [review]

**Comment by Linkid** [comment]
I've just closed some related issues. Thanks for this important PR!

**Pull request merged**
